docs/extraction_quickstart.rst

```diff
@@ -25,6 +25,10 @@ of the available trace classes)::
 
   trace = specreduce.tracing.FlatTrace(image, 15)
 
+.. note::
+  The fit for `~specreduce.tracing.FitTrace` may be adversely affected by noise where the spectrum
+  is faint. In these areas we recommend using a narrow cross-dispersion window to improve the fit.
```

Comment:
"In these areas we recommend using a narrow cross-dispersion window to improve the fit."
-->
"Narrowing the  window parameter or lowering the order of the fitting function may improve the result for noisy data."

Reply:
Changed! Thanks for the suggestion.